src/entry_points/fill.py

```diff
@@ -1,14 +1,56 @@
 """
 Define an entry point wrapper for pytest.
+
+Essentially an alias for pytest but modifies the behavior of --help:
+- 'fill --help' only prints the options defined by ./src/pytest_plugins/.
+- 'fill --help -v' prints the full help (equivalent to 'pytest --help')
+
+The approach is brittle, but pragmatic. There isn't a direct way to trace
+back a command-line option to the plugin that registered it because when
+a plugin registers a command-line option, it doesn't tag that option with
+its own name or ID.
+
+There's an open feature request for this:
+https://github.com/pytest-dev/pytest/issues/9452
 """
 
+import subprocess
 import sys
 
 import pytest
 
 
 def main():  # noqa: D103
-    pytest.main(sys.argv[1:])
+    if "--help" in sys.argv and "-v" not in sys.argv:
+        sys.argv.remove("--help")
+
+        result = subprocess.run(["pytest", "--help"], capture_output=True, text=True)
+        output = result.stdout
+
+        start_string = "Arguments defining evm executable behavior:"
+        end_string = "distributed and subprocess testing:"
```

Comment:
```suggestion
        end_string = "Fill tests until and including the specified fork."
```

Reply:
Thanks for the feedback! Helped me decide to go for a more robust approach, see https://github.com/ethereum/execution-spec-tests/pull/172